### PR TITLE
Updated Readme.md file to include MacOS and Linux OS installation setup and prerequisites

### PR DESCRIPTION
**Pull request by digracesion**
Requirements documentation fix for Issue #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,112 @@ A template for Rails 7, intended to help you quickly build apps for production w
...
+rbenv global 3.2.2
+# set the local version to 3.2.2
+rbenv local 3.2.2
+# set the system version to 3.2.2
+rbenv shell 3.2.2
+```
+
+4. Restart the terminal
+```
+exec zsh
```
Comment (Naokimi) [inline README.md:37]: this assumes having installed zsh too, which is not mentioned in the prerequisites (and probably shouldn't be)
probably best to reword into something like "If you are using Zsh you can run `exec zsh`"

**Review by Naokimi — COMMENTED** [review]
would prefer if the step-by-step instructions were in separate files so as to not make the Readme file too long

**Comment by digracesion** [comment]
Okay, sure. Should I make a separate file for each OS?

**Comment by Naokimi** [comment]
> Okay, sure. Should I make a separate file for each OS?

yes please

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+source 'https://rubygems.org'
```
Comment (Naokimi) [inline Gemfile:1]: probably mistakenly committed?

Comment (digracesion) [reply Gemfile:1]: Right, not really sure how the Gemfile was included in the commit, might have added it by accident 😅

Comment (Naokimi) [reply Gemfile:1]: please remove and merge the commits, then I will approve and merge the PR :)

File: template.rb [hunk 3]
```diff
@@ -3,64 +3,43 @@
 modify rubocop rule to ignore toplevel documentation
 =end
 
-supported_version = 7
+supported_version = ENV['SUPPORTED_RAILS_VERSION'].to_i
```
Comment (Naokimi) [inline template.rb:6]: again accidental commit 😂

**Review by Naokimi — APPROVED** [review]
thank you for your contribution 🙏

**Pull request merged**